R/get_vcov.R

```diff
@@ -15,6 +15,42 @@ get_vcov <- function(model, vcov = NULL, conf_level = NULL, ...) {
     "The `vcov` argument accepts a variance-covariance matrix, a vector of standard errors, or a ",
     "function that returns one of these, such as `stats::vcov`.")
 
+  ## fixest models
+  if (class(model) %in% c("fixest", "fixest_multi")) {
```

Comment:
My guess is the test failures are due to this line, because `class(model)` is a vector with two elements for these classes. So I think you need `class(model)[1] %in% ...` or maybe `inherits(model, "fixest") || inherits(model, "fixest_multi")`